src/pages/RFOX/components/Overview/StakingInfo.tsx

```diff
@@ -113,7 +113,7 @@ export const StakingInfo: React.FC<StakingInfoProps> = ({
 
   return (
     <Box>
-      <Flex alignItems='center' justifyContent='space-between' mb={6}>
+      <Flex flexDir='column' gap={4} mb={6}>
```

Comment:
This is producing an unwanted behavior, the apy's card and values should be row and then column on mobile

See the current app:
<img width="686" alt="image" src="https://github.com/user-attachments/assets/892d1b8a-f5d6-44cd-98b8-d7e0b8515d51" />

vs the current state of this branch:

<img width="591" alt="image" src="https://github.com/user-attachments/assets/4d3b92f6-7e85-4c02-9a3d-74b49dfc59ae" />

```suggestion
      <Flex flexDir={containerFlexDir} gap={4} mb={6}>
```

And the proper chakra responsive value associated so it adapts depending on the viewport width